yoti/com_yoti.xml

```diff
@@ -1,34 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <extension type="component" version="3.0" method="upgrade">
-    <name>Yoti Connect</name>
-    <author>Simon Tong</author>
-    <authorEmail>simon.tong@yoti.com</authorEmail>
+    <name>com_yoti</name>
+    <author>Yoti Ltd</author>
+    <authorEmail>sdksupport@yoti.com</authorEmail>
     <authorUrl>http://www.yoti.com</authorUrl>
     <creationDate>August 2016</creationDate>
     <version>1.0.0</version>
     <license>Link to licence url</license>
     <copyright>Copyright (C) 2016 http://www.yoti.com. All rights reserved.</copyright>
-    <description>Let Yoti users quickly register on your site. Note: Need to enable module in module manager</description>
+    <description>Let Yoti users quickly register on your site. Note: Need to enable Yoti module and Yoti plugin.</description>
```

Comment:
The Yoti module and Yoti plugin will need to be enabled? or You'll need to enable...